src/test/java/com/foundation/salesforce/steps/ContactAcceptanceSteps.java

```diff
@@ -0,0 +1,268 @@
+    public void userGetAStatusCodeWithFirstNameEmailTitle(String statusCode) {
+        Assert.assertEquals(response.getStatusCode(), Integer.parseInt(statusCode));
+    }
+
+    /**
+     * Method to let delete a user existing..
+     */
+    @When("User deletes an existing contact by Id")
+    public void userDeletesAnExistingContactById() {
+        response = requestManager.delete(EndPoints.CONTACT_ENDPOINT + "/" + "00Q3i000002MRlqEA1");
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to le Delete a contact by id.
+     *
+     * @param arg0
+     */
+    @When("User deletes a Contact by Id (.*)")
+    public void userDeletesAContactByIdQIMRlqEAG(int arg0) {
+        response = requestManager.delete(EndPoints.CONTACT_ENDPOINT + "/" + arg0);
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to led find an existing contact by id.
+     */
+    @When("User finds an existing Contact by Id")
+    public void userFindsAnExistingContactById() {
+        response = requestManager.get(EndPoints.CONTACT_ENDPOINT + "/" + "");
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to led find a contact by id.
+     *
+     * @param arg0
+     */
+    @When("User finds a contact by Id (.*)")
+    public void userFindsAContactByIdQIMKLeEA(int arg0) {
+        response = requestManager.get(EndPoints.CONTACT_ENDPOINT + "/" + arg0);
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to le retrieves the summary for a contact.
+     */
+    @When("User retrieves the summary for Contact")
+    public void userRetrievesTheSummaryForContact() {
+        response = requestManager.get(EndPoints.CONTACT_ENDPOINT);
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to le verify status code.
+     *
+     * @param statusCode
+     */
+    @Then("The status code is (\\d+)")
+    public void verify_status_code(int statusCode) {
+        Assert.assertEquals(response.getStatusCode(), statusCode);
+    }
+
+    /**
+     * Method to let set build specifications with the last name.
+     *
+     * @param inputFields
+     */
+    @Given("User sets the lastName")
+    public void userSetsTheLastName(Map<String, String> inputFields) {
+        requestManager.buildSpec(inputFields);
+    }
+
+    /**
+     * Method to update the existing the contact by id.
+     */
+    @When("User updates existing contact by Id")
+    public void userUpdatesExistingContactById() {
+        response = requestManager.patch(EndPoints.CONTACT_ENDPOINT + "/" + "");
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to let set the last name to the contact.
+     *
+     * @param inputFields
+     */
+    @Given("User sets lastName to the contact")
+    public void userSetsLastNameToTheContact(Map<String, String> inputFields) {
+        requestManager.buildSpec(inputFields);
+    }
+
+    /**
+     * Method to let updates contact by id.
+     *
+     * @param arg3
```

Comment:
Please use an appropriate name for this input parameter.
Please, place descriptions for input parameters.